deployments/deploy-pxb-cloud.sh

```diff
@@ -693,6 +693,8 @@ spec:
       value: "${S3_ENCRYPTION_POLICY}"
     - name: USE_GLOBAL_RULES
       value: "${USE_GLOBAL_RULES}"
+    - name: USE_DOCKER_IO
+      value: "${USE_DOCKER_IO}"
```

Comment:
Can we make this `CUSTOM_REGISTRY` and pass the actual string like `docker.io`?
In case we need to test this with some other registry in future it will be very helpful and this will be a one time change. What say?

Reply:
Done